CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
 
 Changes since 1.4.0-rc.1
 
+- Allow overriding the build architecture with --arch, to build
+  images for another architecture than the current host arch.
```

Comment:
Also mention the `--arch-variant` option, and that using `--arch` requires the host to be set up with binfmt_misc.
Surround options with backquotes.

Is the variant only relevant to builds, or if we add it for builds do we need it also for action commands?

Reply:
I think it depends if it needs to run anything, if the sif file is missing it succeeds but not for an overwrite?

`FATAL:   While checking build target: while determining 'alpine.sif' format: the image's architecture (arm64) could not run on the host's (amd64)`

The output from the first successful run does not actually run without fixing the host, though (as expected)

`FATAL:   could not open image /home/anders/apptainer/alpine.sif: the image's architecture (arm64) could not run on the host's (amd64)`

Reply:
Actually I don't know why `apptainer inspect` fails, I didn't expect that command to try to run anything?

Reply:
The only thing that I added was a warning for the previous scenario, that caused the wrong metadata.